ext/storage_sources/gcp_store/README.md

```diff
@@ -0,0 +1,87 @@
+# WiredTiger's GCP Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored
+in Google Cloud Storage using WiredTiger’s provided internal abstraction for storing data in an
+object storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the GCP extension enabled.
+
+### Requirements
+* CMake 3.11 or higher
+* G++ 8.4 or higher
+* Abseil LTS 20230125
+* nlohmann_json library 3.11.2
+* crc32c 1.1.2
+* clang 6.0.0 or higher
+
+### Building
+There is currently only 1 way to build WiredTiger with GCP extension:
+This way manages the GCP SDK dependency as an external project. This method will download the
+GCP SDK, and link to the WiredTiger's build system and build the extension.
```

Comment:
A couple of grammar nitpicks here:
* "to build WiredTiger with GCP extension" -> "to build WiredTiger with the GCP extension"
* "This method will download the
GCP SDK, and link to the WiredTiger's build system and build the extension." -> "This method will download the
GCP SDK, link to WiredTiger's build system, and build the extension."

Reply:
amended https://github.com/wiredtiger/wiredtiger/pull/8803/commits/bb6fc998d9d2d1d3400f2095c7313a57d481ee8c